src/turtle.rs

```diff
@@ -206,6 +206,130 @@ impl Turtle {
         block_on(self.turtle.wait(secs))
     }
 
+    /// Draw a circular arc starting at the current position and going to the left of the turtle,
+    /// thus globally turning counterclockwise.
+    ///
+    /// It is configured through the `radius` and `extent` arguments:
+    ///  * `radius` places the center of the arc itself `Distance` units away from the left of the
+    ///    turtle, with respect to its current orientation. When negative, it does so to the right.
+    ///  * `extent` controls how much of the arc is to be drawn, that is to say the `Angle` that
+    ///    forms the circular portion of the given `radius`. When negative, the turtle moves
+    ///    backwards instead, but it still goes to its left. It can also be greater than the
+    ///    turtle's current angle unit domain limit, i.e. 360° when using degrees or 2π when using
+    ///    radians: the turtle will simply continue to draw until the complete angle is reached.
+    ///
+    /// This method does *nothing* if either one of the provided arguments is not "normal" in the
+    /// sense of [floating-point numbers' definition](f64::is_normal) of it.
+    ///
```

Comment:
```suggestion
```

Let's not document this for now. It's not documented anywhere else and I haven't actually decided yet if "silently fail" is the right way to go here. Explicitly documenting the behaviour is a bit more of a commitment than I want to make right now.

Reply:
Okay, sure.